_data/teamroles.yml

```diff
@@ -109,9 +109,9 @@
 - type: publishing-manager 
   label:
     en: Publishing Manager
-    es: Director/a de publicaciones
+    es: Directora de publicaciones
     fr: Responsable de la publication
-    pt: Gestor/a de publicações
+    pt: Gestora de publicações
```

Comment:
@anisa-hawes, in the case of Managing editors we need to keep both genders because we have 4 different persons sharing that role (Alex, Jenn, Marie and Daniel)

Reply:
Correct @rivaquiroga. Thanks! In this case it should be "Gestor/a"